Cargo.toml

```diff
@@ -55,6 +55,7 @@ hex = "0.4.3"
 indexmap = { version = "2", features = ["std"] }
 itoa = "1.0.6"
 itoap = { version = "1", features = ["simd"] }
+kstring = { version = "2.0.2", features = ["arc", "max_inline", "serde"] }
```

Comment:
> `max_inline`: Instead of aligning the inline-string for performance (15 bytes + length on 64-bit), use the full width (22 bytes on 64-bit)

I chose `max_inline` as I think if we can inline more then we can reduce atomic refcount operations.

https://crates.io/crates/kstring